third-party/realdds/doc/stream-configurations.md

```diff
@@ -0,0 +1,75 @@
+
+See also: [device](device.md)
+
+
+# Stream Configurations
+
+___
+## Infrared
+
+When only a single infrared stream is available, only a single topic is expected.
+
+When multiple streams are available (Left/Right), two topics will be available with each providing the same format and encoding. This is true even if the sensor generates an interlaced format: it is up the server to separate into two separate streams, each distinct.
+
+#### Format
+
+See the [ROS2 Image format](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Image.msg).
+
+#### Encoding
+
+This should be set to `mono8`, corresponding to `Y8` in librealsense. No interlaced formats.
+
+#### Metadata
+
+A single metadata message is expected for all messages from the same sensor (so one for depth, IR left, IR right).
```

Comment:
I think that we send a metadata message per active stream, not per active sensor

Reply:
Right now, yes, you're correct. And after talking to Evgeni, I think I'll take that statement back.